lib/ueberauth/adfs/oauth.ex

```diff
@@ -1,28 +1,18 @@
 defmodule Ueberauth.Strategy.ADFS.OAuth do
   @moduledoc """
-
-  ## OAuth2 for ADFS
-
-  In ADFS >=3.0 setup a new Client using Powershell:
-  ```powershell
-  Add-AdfsClient -Name "OAUTH2 Client" -ClientId "unique-custom-client-id" -RedirectUri "http://localhost:4000/auth/adfs/callback"
-  Add-ADFSRelyingPartyTrust -Name "OAUTH2 Client" -Identifier "http://localhost:4000/auth/adfs"
-  Set-AdfsRelyingPartyTrust -IssuanceAuthorizationRulesFile "TransformRules.txt"
-  ```
-  In TransformRules.txt put the following:
-  ```
-  @RuleTemplate = "LdapClaims"
-  @RuleName = "User Details"
-  c:[Type == "http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname", Issuer == "AD AUTHORITY"]
-  => issue(store = "Active Directory", types = ("http://schemas.microsoft.com/ws/2008/06/identity/claims/windowsaccountname", "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname", "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname", "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress", "groups", "userPrincipalName"), query = ";sAMAccountName,givenName,sn,mail,tokenGroups,userPrincipalName;{0}", param = c.Value);
-  ```
-
-  Add `adfs_url` and `client_id` to your configuration:
-  ```elixir
-  config :ueberauth, Ueberauth.Strategy.ADFS.OAuth,
-    adfs_url: System.get_env("ADFS_URL"),
-    client_id: System.get_env("ADFS_CLIENT_ID")
-  ```
+  OAuth2 for ADFS
+  Add 'adfs_url', 'adfs_metadata_url', 'client_id' and 'resource_identifier'
+  to your configuration:
+  config :ueberauth, Ueberauth.Strategy.ADFS,
+    adfs_url: "https://adfs.url",
+    adfs_metadata_url: "https://path.to/FederationMetadata.xml",
+    client_id: "the_client",
+    resource_identifier: "the_resource_id"
+
+  On the ADFS server add a new Client:
+  Add-AdfsClient -Name "the_client_name" -ClientId "the_client" -RedirectUri "https://my_application.url/auth/adfs/callback"
+
+  Setup the relying party trust as usual, be sure to return at least the e-mail in the clams.
```

Comment:
Might wanna keep some parts of the original doc here, for people that want to set this up with a powershell script or someting like that. Maybe also Windows Server 2016 instructions, since you should be able to just create a client without powershell there?